### PR TITLE
Quickstart PR with corrected redirect

### PR DESCRIPTION
**Pull request by SaptakS**
##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Refers/Fixes

Refs #1029

### CODE REVIEW

**Comment by mbeacom** [comment]
This is currently live on https://stage.gitcoin.co

Bugs so far:

- [ ] Submitting a bounty fails with simply redirecting back to the funding form
- [ ] Checkboxes use two different styles on the funding page

File: app/assets/v2/css/base.css [hunk 1]
```diff
@@ -1160,6 +1149,27 @@ input.is-invalid {
...
+
+.submit-notification a {
+  color: #fff;
+}
+
+.metamask-error {
+  background-color: #FF6700;
+}
+
+
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: extra line :P

File: app/assets/v2/css/base.css [hunk 2]
```diff
@@ -1160,6 +1149,27 @@ input.is-invalid {
...
+  min-height: 100px;
+}
+
+.submit-notification h3 {
+  font-size: 20px !important;
+  text-transform: capitalize;
+}
+
+.submit-notification a {
+  color: #fff;
```
Comment (thelostone-mc) [inline app/assets/v2/css/base.css]: Don't hate me for this but can we uppercase this ( to keep it uniform )

Comment (thelostone-mc) [reply app/assets/v2/css/base.css]: or lower case everything! we'll have work either way

**Comment by thelostone-mc** [comment]
LGTM left a few minor comments

**Pull request merged**